mobile/apps/partner_profile/test/templates.coffee

```diff
@@ -9,6 +9,20 @@ Partner = require '../../../models/partner'
 Artists = require '../../../collections/artists'
 fixtures = require '../../../test/helpers/fixtures'
 
+INSTITUTION        = 'institution'
+GALLERY_DEFAULT    = 'gallery_default'
+GALLERY_DEPRECATED = 'gallery_deprecated'
+GALLERY_ONE        = 'gallery_one'
+GALLERY_TWO        = 'gallery_two'
+GALLERY_THREE      = 'gallery_three'
+
+PARTNER_TYPES = [
+  INSTITUTION
+  GALLERY_ONE
+  GALLERY_TWO
+  GALLERY_THREE
+]
```

Comment:
I don't feel too strongly about either way. Including it in a model feels more conventional, but technically it has less to do with the actual "profile", but more about mix of partner type, subscription plan, and potentially future business logic.

Putting it in sharify data would be included on every page, if I understand correctly, and could be an overhead, but maybe worth it if it's used in more places. Also, it may be convenient to be able to control the values in env vars rather than hard code them.